Public/FourKeyMetrics.ps1

```diff
@@ -167,14 +173,30 @@ function global:Get-ReleaseMetrics {
             Write-Warning "Release $($thisRelease.TagRef) has no relevant commits and will be ignored"
         }
         else {
-            [PSCustomObject]@{
-                From             = $previousRelease.TagRef;
-                To               = $thisRelease.TagRef;
-                FromDate         = $previousRelease.Date;
-                ToDate           = $thisRelease.Date;
-                Interval         = $thisRelease.Date - $previousRelease.Date;
-                IsFix            = $thisRelease.IsFix;
-                CommitAges       = $CommitAges;
+            if ($componentName -ne ""){
```

Comment:
I wonder if this complexity is necessary? What harm would it do to always set the `Component`?

Reply:
None, I'll remove the condition and just always add component name thanks